libs/full/async_cuda/include/hpx/async_cuda/cuda_exception.hpp

```diff
@@ -25,10 +25,6 @@ namespace hpx { namespace cuda { namespace experimental {
           , err_(err)
         {
         }
-        cudaError_t get_cuda_errorcode()
-        {
-            return err_;
-        }
```

Comment:
Why?

Reply:
I was unused but as you explained it can be useful